GITHUB_ORG_MANGEMENT_POLICY.md

```diff
@@ -0,0 +1,93 @@
+# OpenJS GitHub Organization Management Policy
+
+The OpenJS Foundation GitHub Organization (https://github.com/openjs-foundation) is
+provided as a development resource by the OpenJS Foundation under the direction
+of the OpenJS Cross Project Council (CPC).
+
+## OpenJS Admin Repository
```

Comment:
Should we have an `/admin` repo or should we simply use the CPC repo?

Reply:
We discussed this [when the issue was created](https://github.com/openjs-foundation/cross-project-council/issues/185) during the bootstrap phase, and decided to not create an `/admin` repo at that point since there is only one top level council who's responsible for administrative tasks. That's still the case even though we have a few more repos in the org now, and I don't see a need for an admin specific repo yet – though we could spin one out when we have another top level committee with administrative power. Keeping these docs in the CPC repo until then sounds good to me. 😄

Reply:
I don't think we need a separate admin repo